.chloggen/add-autogen-equal-pdata.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: pdata
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Introduce auto-generated equal methods for pdata
```

```suggestion
note: Introduce `Equal` methods for objects comparison.
```

We should also have a changelog entry for pprofile, which is still its own module.